onnxruntime/core/framework/bfc_arena.cc

```diff
@@ -101,7 +101,8 @@ bool BFCArena::Extend(size_t rounded_bytes) {
 
       extend_bytes = std::min(curr_region_allocation_bytes_, available_bytes);
 
-      // we allocated the same number of bytes as the current region, so we have 2x that now
+      // we allocated the same number of bytes as the current region
+      // the 2x is to double the minimum size of the next amount we'll allocate
```

Comment:
You'll introduce a bug back. :-( 

Be very careful on line 107. It could easily overflow on 32bit systems. And one of our test model will just produce incorrect result because of that. I reported the bug to Chen Tang and he fixed it. But it look like you will bring it back.

Reply:
I am not changing anything in the code, except re-wording the comments.

Reply:
Sorry I was wrong, I thought it was fixed.